.github/workflows/ci.yml

```diff
@@ -24,8 +25,12 @@ jobs:
         with:
           node-version: 18
 
-      - run: pip install -r src/python-fastui/requirements/all.txt
-      - run: pip install src/python-fastui
+      - name: Install the latest version of rye
+        uses: eifinger/setup-rye@v2
+
+      - run: make install
+
+      - run: make lint
 
       - run: npm install
```

Comment:
I guess we need this for the next few lines, but maybe we could make them part of the same job for clarity?

Reply:
sorry didn't catch that. what do you mean?

Reply:
Never mind - not a big deal!